skyportal/handlers/api/internal/ephemeris.py

```diff
@@ -6,26 +6,29 @@
 
 class EphemerisHandler(BaseHandler):
     @auth_or_token
-    def get(self, telescope_id):
```

Comment:
Maybe we should do it like the others where telescope_id=None by default but also allow to get ephemederies from just 1 by ID, and otherwise use your multiple trick?